lib/modules/dosomething/dosomething_auto_translate/dosomething_auto_translate.module

```diff
@@ -0,0 +1,88 @@
+<?php
+
+/**
+ * @file dosomething_auto_translate.module
+ * This module uses the Google Translate API to translate campaign nodes.
+ */
+
+/**
+ * @file dosomething_auto_translate.module
+ *
+ */
+function dosomething_auto_translate_campaigns($start = NULL, $limit = NULL) {
+  $query = new EntityFieldQuery();
+
+  $query->entityCondition('entity_type', 'node')
+    ->entityCondition('bundle', 'campaign')
+    ->propertyCondition('status', NODE_PUBLISHED)
+    ->range($start, $limit);
```

Comment:
will this break if no values are passed? maybe set them to numbers instead of `NULL` by default?